Update nimbus.md

BreadandCinnamon opened the pull request:
multiple spelling errors and some wrong information.

BreadandCinnamon committed:
Update nimbus.md

multiple spelling errors and some wrong information.

BTKingly commented:
Since amactus was the one to last edit this page, I'm giving him the benefit of the doubt that the information is correct, thus there's no need to change it.

BTKingly committed:
Update nimbus.md

- Reverted nimbus back to original, with the exception of the spelling error that we discussed
- Also took advantage of this to add discord rank
